core/lib/dr_inject.h

```diff
@@ -112,10 +112,11 @@ DR_EXPORT
  * \param[out]  data           An opaque pointer that should be passed to
  *                             subsequent dr_inject_* routines to refer to
  *                             this process.
+ * \param[out]  app_name       pointer to the name of the target process
```

Comment:
style: Capitalize and punctuate user-facing documentation sentences.

Reply:
This should point out that this pointer is only valid until dr_inject_process_exit.

Reply:
Done